lib/integration/queryManager.js

```diff
@@ -0,0 +1,34 @@
+import { logMessage } from "../logger";
+import { dbConnector } from "./dbConnector";
+
+export const executeQuery = async (sql, params) => {
+  const client = await dbConnector();
+  return client
```

Comment:
fix mixed async await and promise stying.

Should be 

`try{
     const data = await client.query()
}catch(err){
 ...
}`